website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,77 @@
 
-These are the following options for setting up email notifications. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for info on license types eligible for email notifications. 
+### Configure email notifications
 
-- As a **user** &mdash; You can set up email notifications for yourself under your Profile.
-- As an **admin** &mdash; You can set up notifications on behalf of your team members. 
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. By default, the **Notification email** option is set to the email address that's in [your dbt Cloud personal profile](https://cloud.getdbt.com/settings/profile). 
 
-To set up job notifications, follow these steps:
+    If you're an **admin**, you can choose a different email address. Select the **Notification email** dropdown and choose another address from the list. The list includes internal users with access to the account and also external emails that have been added. 
+    - To add an external email address, select the **Notification email** dropdown and choose **Add external email**. After you add the external email, it becomes available for selection in the **Notification email** dropdown list.
 
-1. Click the gear menu in the top right corner and select **Notification Settings**.
+    <Lightbox src="/img/docs/deploy/example-notification-external-email.png" width="50%" title="Example of the Notification email dropdown"/>
 
-2. Select **Edit** to begin editing the **Email Notifications** settings.
-    - **As a user:** Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want to receive notifications for.
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
 
-    - **As an admin:**  Under **Configure notifications for**, use the dropdown to select one or more users you'd like to set notifications for. If you only see your own name, then you might not have admin privileges. <br /><br />
-    Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want them to receive notifications for.
+1. Click **Edit** to configure the email notification settings.
 
-3. Click **Save**.
+    - If you're a **user**, choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications for.
 
-    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/email-notifications.png" width="75%" title="Configuring Email Notifications"/>
+    - If you're an **admin**, use the **Configure notifications for** dropdown to select all the users you want to sent notifications to. Then, choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want them to receive notifications for.
```

Comment:
I don't think this is totally accurate. You can only configure notifications for one user at a time, and the clicking edit disables the dropdown until the currently selected item has been saved

Reply:
Can we also specify "account admin" here?